examples/IrisClassification/iris_classification.py

```diff
@@ -124,4 +218,17 @@ def test_step(self, batch, batch_idx):
     trainer.fit(model, dm)
     trainer.test()
 
-    torch.save(model.state_dict(), "iris.pt")
+    input_schema = Schema(
+        [
+            ColSpec("double", "sepal length (cm)"),
+            ColSpec("double", "sepal width (cm)"),
+            ColSpec("double", "petal length (cm)"),
+            ColSpec("double", "petal width (cm)"),
+        ]
+    )
+    output_schema = Schema([ColSpec("long")])
+    signature = ModelSignature(inputs=input_schema, outputs=output_schema)
+    with mlflow.start_run():
+        if os.path.exists("model"):
+            shutil.rmtree("model")
+        mlflow.pytorch.save_model(trainer.get_model(), "model", signature=signature)
```

Comment:
```suggestion
        mlflow.pytorch.log_model(trainer.get_model(), "model", signature=signature)
```

Can we use `log_model` here?

Reply:
By default, torchserve only copies the model file,  handler file from the mlflow model. For inferring the signature MLmodel file is needed and the file needs to be passed as an `extra_file` argument to `torch-model-archiver`.

It would be more easier if we use `mlflow.pytorch.save_model` , as user can reference the MLmodel file path as `model/MLmodel`

In case of log_model, there might be multiple runs and it would be difficult for the user to find the MLmodel path. And that is the reason we preferred save_model here in this example

What do you think ?